### PR TITLE
Fixup places where it should be CCM instead of GCM.

### PR DESCRIPTION
**Pull request by anhu**
Fixes https://github.com/wolfSSL/wolfssl/issues/7216

### CODE REVIEW

File: src/keys.c [hunk 1]
```diff
@@ -1440,7 +1440,7 @@ int GetCipherSpec(word16 side, byte cipherSuite0, byte cipherSuite,
         specs->static_ecdh           = 0;
         specs->key_size              = SM4_KEY_SIZE;
         specs->block_size            = SM4_BLOCK_SIZE;
-        specs->iv_size               = GCM_IMP_IV_SZ;
```
Comment (SparkiDev) [inline src/keys.c:1443]: If you change this, then change the other places for AES CCM too.

File: wolfssl/internal.h [hunk 2]
```diff
@@ -1743,6 +1743,7 @@ enum Misc {
     AES_CCM_16_AUTH_SZ  = 16, /* AES-CCM-16 Auth Tag length */
     AES_CCM_8_AUTH_SZ   = 8,  /* AES-CCM-8 Auth Tag Length  */
     AESCCM_NONCE_SZ     = 12,
+    CCM_IMP_IV_SZ       = 4,  /* Size of GCM/CCM AEAD implicit IV */
```
Comment (SparkiDev) [inline wolfssl/internal.h:1746]: Fix the comment on this to be only CCM.
Fix the comment for GCM_IMP_IV_SZ to be only GCM.

Comment (dgarske) [reply wolfssl/internal.h:1746]: Please fix comment. Then we can accept this PR.

**Review by dgarske — CHANGES_REQUESTED** [review]

**Review by dgarske — APPROVED** [review]
This PR doesn't appear to fix anything, but is an acceptable cleanup. Over to @SparkiDev for final.

**Comment by anhu** [comment]
> This PR doesn't appear to fix anything, but is an acceptable cleanup. Over to @SparkiDev for final.

Yes, was in response to a github bug report.

**Comment by anhu** [comment]
jenkins retest this please

**Review by SparkiDev — APPROVED** [review]

**Pull request merged**